.teamcity/settings.kts

```diff
@@ -6,7 +6,7 @@ import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
 version = "2021.2"
 
 project {
-    vcsRoot(RhinoLicensingVcs)
+    vcsRoot(DslContext.settingsRoot)
```

Comment:
No action to take here just yet (will do a quick test), but this line _can_ be removed (but not sure if it _needs_ to be removed or not) will report back.